### PR TITLE
make inputFields a list of lists + change double quotes + some fixes

### PR DESCRIPTION
**Pull request by moranraz**
Fixes #

  - changed inputFields to a list of lists, where every sublist is a set of sufficient fields to run the query. 
  - changed all double quotes to single quotes (to allow better handling of the queries in our code).
  - made a few changes in the queries.

### CODE REVIEW

File: Exploration Queries/InputEntity_Host/Host2Acc_PossibleSuccessfulBruteForce.txt [hunk 1]
```diff
@@ -14,22 +14,22 @@ let SuccessfulLoginEventId = 4624;
 let FailedLoginEventId = 4625;
 let AccountsPossibleSuccessfulBruteForce = (v_Host_HostName:string, v_Host_NTDomain:string, v_Host_DnsDomain:string){
 SecurityEvent
-	| where AccountType == "User"
+	| where AccountType == 'User'
 		and tolower(Computer) startswith tolower(v_Host_HostName) 
 		and (
-			tolower(Computer) contains tolower(v_Host_NTDomain) or
-			tolower(Computer) endswith tolower(v_Host_DnsDomain)
+			(v_Host_NTDomain != '' and tolower(Computer) contains tolower(v_Host_NTDomain)) or
```
Comment (shainw) [inline Exploration Queries/InputEntity_Host/Host2Acc_PossibleSuccessfulBruteForce.txt]: Any reason you chose not to use isempty(v_Host_NTDomain) here versus v_Host_NTDomain != '' ?

Comment (shainw) [reply Exploration Queries/InputEntity_Host/Host2Acc_PossibleSuccessfulBruteForce.txt]: I meant isnotempty() :)

File: Exploration Queries/InputEntity_Host/Host2Acc_PossibleSuccessfulBruteForce.txt [hunk 2]
```diff
@@ -14,22 +14,22 @@ let SuccessfulLoginEventId = 4624;
...
+			(v_Host_DnsDomain != '' andtolower(Computer) endswith tolower(v_Host_DnsDomain))
```
Comment (shainw) [inline Exploration Queries/InputEntity_Host/Host2Acc_PossibleSuccessfulBruteForce.txt]: Same here on isnotempty(v_Host_NTDomain) also there is no space between "and" and "tolower(Computer)"

File: Exploration Queries/SigninLogs/LeastPrevUsersByIPAddress.txt [hunk 3]
```diff
@@ -16,11 +16,11 @@ SigninLogs
...
+| extend info = pack('AppDisplayName', AppDisplayName, 'ClientAppUsed', ClientAppUsed, 'Browser', tostring(Browser), 'IPAddress', IPAddress, 'ResultType', ResultType, 'ResultDescription', ResultDescription, 'Location', Location, 'State', State, 'City', City, 'StatusCode', StatusCode, 'StatusDetails', StatusDetails)
 | summarize min(TimeGenerated), max(TimeGenerated), count(), Account_Aux_info = makeset(info) by RemoteHost , UserDisplayName, tostring(OS), UserPrincipalName, AADTenantId, UserId
 | top 10 by count_ asc nulls last 
 | project StartTimeUtc = min_TimeGenerated, EndTimeUtc = max_TimeGenerated, RemoteHost, UserDisplayName, OS, UserPrincipalName, AADTenantId, UserId, Account_Aux_info
-| project-rename Account_UnstructuredName=UserPrincipalName, Account_DisplayName=UserDisplayName, Account_AadTenantId=AADTenantId, Account_AadUserId=UserId, Account_Host_UnstructuredName=RemoteHost, Account_Host_OsVersion=OS
+| project-rename Account_UnstructuredName=UserPrincipalName, Account_DisplayName=UserDisplayName, Account_AadTenantId=AADTenantId, Account_AadUserId=UserId, Account_Host_UnstructuredName=RemoteHost, Account_Host_OSVersion=OS
```
Comment (shainw) [inline Exploration Queries/SigninLogs/LeastPrevUsersByIPAddress.txt]: Was this change intended? Account_Host_**OS**Version=OS

File: Exploration Queries/SigninLogs/MostPrevUsersByIPAddress.txt [hunk 4]
```diff
@@ -16,11 +16,11 @@ SigninLogs
...
+| extend info = pack('AppDisplayName', AppDisplayName, 'ClientAppUsed', ClientAppUsed, 'Browser', tostring(Browser), 'IPAddress', IPAddress, 'ResultType', ResultType, 'ResultDescription', ResultDescription, 'Location', Location, 'State', State, 'City', City, 'StatusCode', StatusCode, 'StatusDetails', StatusDetails)
 | summarize min(TimeGenerated), max(TimeGenerated), count(), Account_Aux_info = makeset(info) by RemoteHost , UserDisplayName, tostring(OS), UserPrincipalName, AADTenantId, UserId
 | top 10 by count_ desc nulls last 
 | project StartTimeUtc = min_TimeGenerated, EndTimeUtc = max_TimeGenerated, RemoteHost, UserDisplayName, OS, UserPrincipalName, AADTenantId, UserId, Account_Aux_info
-| project-rename Account_UnstructuredName=UserPrincipalName, Account_DisplayName=UserDisplayName, Account_AadTenantId=AADTenantId, Account_AadUserId=UserId, Account_Host_UnstructuredName=RemoteHost, Account_Host_OsVersion=OS
+| project-rename Account_UnstructuredName=UserPrincipalName, Account_DisplayName=UserDisplayName, Account_AadTenantId=AADTenantId, Account_AadUserId=UserId, Account_Host_UnstructuredName=RemoteHost, Account_Host_OSVersion=OS
```
Comment (shainw) [inline Exploration Queries/SigninLogs/MostPrevUsersByIPAddress.txt]: Was this change intended? Account_Host_**OS**Version=OS

File: Exploration Queries/InputEntity_Account/Acc2Host_HostWithMostFails.txt [hunk 5]
```diff
@@ -16,19 +16,21 @@ let MostFailedLogins = (v_Account_Name:string, v_Account_NTDomain:string, v_Acco
...
+	| extend NameAndUPNSuffix =  split(Account,""@"") 
+	| extend NameAndNtDomain = split(Account,""\\"")
+    | extend Account_UPNSuffix = iff(array_length(NameAndUPNSuffix) == 2, NameAndUPNSuffix[1], """")
+    | extend Account_NTDomain = iff(array_length(NameAndNtDomain) == 2, NameAndNtDomain[0], """")
+    | extend Account_Name = iff(array_length(NameAndNtDomain) == 2, NameAndNtDomain[1], iff(array_length(NameAndUPNSuffix) == 2, NameAndUPNSuffix[0], Account))
 	// filter by account
-	| where (Account_Name != "" and tolower(Account_Name) == tolower(v_Account_Name)) 
-		and ((Account_NTDomain != "" and tolower(v_Account_NTDomain) == tolower(Account_NTDomain) ) or 
-			 (Account_UPNSuffix != "" and tolower(v_Account_UPNSuffix) == tolower(Account_UPNSuffix))
+	| where (Account_Name != '' and tolower(Account_Name) == tolower(v_Account_Name))
```
Comment (laithhisham) [inline Exploration Queries/InputEntity_Account/Acc2Host_HostWithMostFails.txt]: isnotempty(Account_Name)
